### PR TITLE
qa: add basic omap testing capability

### PR DESCRIPTION
**Pull request by neha-ojha**
This will let us run omap workloads using the radosbench task.

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by neha-ojha** [comment]
Test results: http://pulpito.ceph.com/nojha-2019-07-18_18:17:55-rados:perf-master-distro-basic-smithi/

**Comment by neha-ojha** [comment]
retest this please

File: qa/suites/rados/perf/workloads/radosbench_omap_write.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+tasks:
+- radosbench:
+    clients: [client.0]
+    write-omap: True
+    time: 300
```
Comment (jdurgin) [inline qa/suites/rados/perf/workloads/radosbench_omap_write.yaml]: could you add the io/object size explicitly, so it's clear here and doesn't change if the defaults change in the future

Comment (neha-ojha) [reply qa/suites/rados/perf/workloads/radosbench_omap_write.yaml]: we default to 65536, as per the task https://github.com/ceph/ceph/blob/master/qa/tasks/radosbench.py#L79

**Comment by tchaikov** [comment]
jenkins, retest this please

**Comment by neha-ojha** [comment]
http://pulpito.ceph.com/nojha-2019-07-19_20:19:04-rados:perf-master-distro-basic-smithi/

File: qa/suites/rados/perf/workloads/radosbench_omap_write.yaml (hunk 1 above)
Comment (neha-ojha) [reply qa/suites/rados/perf/workloads/radosbench_omap_write.yaml]: also added `size` (op_size)

**Pull request merged**